scripts/deploy-on-openshift.sh

```diff
@@ -24,4 +24,5 @@ oc apply -f manifests/cluster-monitoring-operator-role.yaml
 oc apply -f manifests/cluster-monitoring-operator-role-binding.yaml
 oc apply -f manifests/cluster-monitoring-config.yaml
 oc apply -f manifests/secret-etcd-certs.yaml
-oc apply -f manifests/cluster-monitoring-operator.yaml
\ No newline at end of file
+oc apply -f manifests/cluster-monitoring-operator.yaml
+oc apply -f manifests/cluster-monitoring-operator-service.yaml
```

Comment:
we need to deploy a service so we can monitor the CMO

Reply:
maybe we should put this in the task as well? I'd prefer not to have to add this to openshift-ansible or whatever is going to deploy cluster-monitoring in the future